extensions/2.0/Khronos/KHR_materials_anisotropy/README.md

```diff
@@ -0,0 +1,140 @@
+# KHR\_materials\_anisotropy
+
+## Contributors
+
+* Jim Eckerlein, Norbert Nopper, UX3D
+
+## Status
+
+Suggestion
+
+## Dependencies
+
+Written against the glTF 2.0 spec.
+
+## Overview
+
+This extension defines the anisotropic property of a material as observable with brushed metals for instance.
+An asymetric specular lobe model is introduced to allow for such phenomena.
+The visually distinct feature of that lobe is the elongated appearance of the specular reflection.
+For a single punctual light source, the specular reflection will eventually degenerate into a zero width line in the limit,
+that is where the material is fully anisotropic, as opposed to be fully isotropic in which case the specular reflecation is radially symmetric.
+
+## Extending Materials
+
+Sample values:
+
+```json
+{
+    "materials": [
+        {
+            "extensions": {
+                "KHR_materials_anisotropy": {
+                    "anisotropy": 0.6,
+                    "anisotropyDirection": [0.0, 1.0, 0.0]
```

Comment:
Why do we need a separate anisotropy direction vector? We already define a tangent/bitangent direction and if positive anisotropy is in the tangent direction and negative is in the bitangent direction, isn't that enough?